be/src/vec/exprs/table_function/table_function_factory.cpp

```diff
@@ -65,22 +69,26 @@ const std::unordered_map<std::string, std::function<std::unique_ptr<TableFunctio
                 {"explode_map", TableFunctionCreator<VExplodeMapTableFunction> {}},
                 {"explode", TableFunctionCreator<VExplodeTableFunction> {}}};
 
-Status TableFunctionFactory::get_fn(const std::string& fn_name_raw, ObjectPool* pool,
-                                    TableFunction** fn) {
-    bool is_outer = match_suffix(fn_name_raw, COMBINATOR_SUFFIX_OUTER);
-    std::string fn_name_real =
-            is_outer ? remove_suffix(fn_name_raw, COMBINATOR_SUFFIX_OUTER) : fn_name_raw;
+Status TableFunctionFactory::get_fn(const TFunction& t_fn, ObjectPool* pool, TableFunction** fn) {
+    const std::string fn_name_raw = t_fn.name.function_name;
```

Comment:
should not be call before if，because semms useless in if {}.

and use `std::string_view` or reference